lib/Common/Codex/Utf8Helper.h

```diff
@@ -69,8 +75,10 @@ namespace utf8
         }
 
         utf8::DecodeIntoAndNullTerminate(destString, (LPCUTF8) sourceString, cchDestString);
+        Assert(destString[cchDestString] == 0);
```

Comment:
I am trying to do a minimum validation of result WCHAR string length because this returns the length now. `cchDestString` was calculated by another API. This confirms a bit if `DecodeIntoAndNullTerminate` did decode these many characters.